src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -21,6 +21,11 @@ public static partial class Kernel32
         /// </summary>
         public const int PIPE_UNLIMITED_INSTANCES = 255;
 
+        /// <summary>
+        /// All processes start at this shutdown level
+        /// </summary>
+        public const int DEFAULT_SHUTDOWN_LEVEL = 0x280;
```

Comment:
Makes sense, but in such cases IMO we should use PascalCase, like we do when we define `None = 0` for these enums, to make it clear we added it.